integration/configs/browserCompatibilityTest/linux/linux_content_share_only_allow_two_test.config.json

```diff
@@ -0,0 +1,26 @@
+{
+    "`": 1,
+    "name": "linux_content_share_only_allow_two_test.json",
+    "grids": [
+    ],
+    "tests": [
+      {
+        "name": "Content Share Only Allow Two Test on Linux %ts",
+        "description": "Content Share Only Allow Two Max on Linux",
```

Comment:
It's better to say: "Only allow up to two content shares at the same time on Linux"

Reply:
Done